Deployment & readme

hankaEsha opened the pull request.

hankaEsha committed:
Responsive images added

hankaEsha committed:
Deployment service changed from surge.sh to vercel.com, information about setting up vercel added into README

hankaEsha committed:
Update README.md

benabraham reviewed (changes requested):
Prosím o změny tam, kde je to jasné.
Koncepčně tam chybí nasazení na stejnou „produkční“ adresu pro další členy týmu. Tedy jak to nasadím já? Ne někam k sobě, ale na tu doménu? 
Tedy připojení k projektu, které je myslím popsáno tady: https://vercel.com/docs/cli#introduction/project-linking
A jak se k tomu dostanu já nebo někdo další pak zdokumentovat.

hankaEsha committed:
Typos fixed

Co-authored-by: Daniel Srb <benabraham@users.noreply.github.com>

hankaEsha committed:
deployment optimised

hankaEsha committed:
Merge branch 'deployment' of https://github.com/hankaEsha/kavarna-ghc into deployment

hankaEsha committed:
1) Images in Coffe section corrected to work in older browsers
2) Images' sizes corrected in HTML for main page, menu page and our story page
Resolves PR #2

hankaEsha committed:
documentation updated

benabraham commented on README.md:
```diff
@@ -53,16 +53,10 @@ npm run build
 
-You don't need to leverage the power of templates. You can just create plain HTML files with `*.njk` extension.
-
-If you don't want a template to be turned into HTML file start it with `_`. Typically these are templates used for _include_ or _extend_.
```
tohle mohlo zůstat

hankaEsha committed:
documentation updated
